roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+openshift_logging_eventrouter_replicas: 1
+openshift_logging_eventrouter_sink: stdout
```

Comment:
We should add the allowable values as a list to vars/main.yaml and validate the choice of sync.  There is a similar pattern in the elasticsearch role for validating the logger outputs (e.g. file, console)